bridge/deploy/bridge.yaml

```diff
@@ -46,6 +46,10 @@ spec:
            value: "true"
          - name: SHOW_API_TOKEN
            value: "true"
+         - name: KEPTN_INSTALLATION_TYPE
+           value: "QUALITY_GATES,CONTINUOUS_OPERATIONS,CONTINUOUS_DELIVERY"
```

Comment:
There is a mismatch! it says `QUALITY_GATES` here, but we check for `QUALITY_GATES_ONLY` in the code above

Reply:
I fixed it! Thanks.